src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```
@@ -511,6 +511,20 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
             string centralPackageTransitivePinningEnabled = GetPropertySafe(_vsProjectAdapter.BuildProperties, ProjectBuildProperties.CentralPackageTransitivePinningEnabled);
             // Do not add new properties here. Use BuildProperties.GetPropertyValue instead, without DTE fallback.
 #pragma warning restore CS0618 // Type or member is obsolete
+            string skdAnalysisLevelString = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.SdkAnalysisLevel);
+            string usingNetSdk = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.UsingMicrosoftNETSdk);
+            bool usingMicrosoftNetSdk = false;
+            NuGetVersion sdkAnalysisLevel = null;
+
+            if (skdAnalysisLevelString != null)
+            {
+                sdkAnalysisLevel = new NuGetVersion(skdAnalysisLevelString);
+            }
+
+            if (bool.TryParse(usingNetSdk, out bool result))
+            {
+                usingMicrosoftNetSdk = result;
+            };
```

What's the error experience when other boolean properties can't be parsed?  Do we silently ignore and use a default, do we warn and use a default, or do we error?